src/index.html

```diff
@@ -4,9 +4,467 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <section
+          class="card catalog__card"
+          data-qa="card"
+        >
+          <img
+            src="../src/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+
+          <h3 class="card__headline">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star">
+              </div>
+
+              <div class="stars__star">
+              </div>
+
+              <div class="stars__star">
+              </div>
+
+              <div class="stars__star">
+              </div>
+
+              <div class="stars__star">
+              </div>
+            </div>
+            <p class="card__reviews-count">
+              Reviews: 5
+            </p>
+          </div>
```

Comment:
I think we need to add an empty line here cause both of this blocks are "card__reviews" childs.
```suggestion
          <div class="card__reviews">
            <div class="stars stars--4">
              <div class="stars__star">
              </div>

              <div class="stars__star">
              </div>

              <div class="stars__star">
              </div>

              <div class="stars__star">
              </div>

              <div class="stars__star">
              </div>
            </div>
            
            <p class="card__reviews-count">
              Reviews: 5
            </p>
          </div>
```